docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3284,17 +3287,19 @@ Limitations:
 
 **Age Miswriting** 
 
-To implement this, first select the strings eligible for noise. For each 
-selected string, the age will be adjusted. The adjustment value will be 
-randomly selected from the user inputted list of possible perturbations. 
+To implement this, first select the rows for noise according to the row noise probability.
+For each selected row, the age will be adjusted. The adjustment value will be 
+randomly selected from the configured list of possible perturbations,
+according to the configured probabilities of selection. 
 
-For example, if the correct age is 28 and the possible perturbations are [-2, -1, 1, 2] 
-then 28 will be adjusted to either: 26, 27, 29, or 30, with an equal chance 
-of each option. 
+For example, if the correct age is 28 and the possible perturbations are [-2, -1, 1, 2]
+with configured probabilities of [0.2, 0.3, 0.3, 0.2]
+then 28 will be adjusted to either: 26, 27, 29, or 30, with a 0.2 probability for each of
+26 and 30 and a 0.3 probability for each of 27 and 29.
 
-If the resulting age is negative (e.g., the correct age is 1 and a perturbation 
-of -2 is applied to make the answer -1), then reselect from the remaining 
-perturbation options until the final answer is 0 or higher. 
+If the age after adding the chosen perturbation is negative, flip the sign to be positive (e.g. a -2 becomes 2).
+If after this sign flip, the resulting age is equal to the original age value, subtract 1 from the age.
```

Comment:
I would be fine with a failed perturbation in some cases, to keep things simple for engineering.  (If this is also considered simple, I'm fine with it, too.)